datajunction-server/alembic/versions/2025_01_22_1638-0e428c1bb9bc_notification_preferences_table.py

```diff
@@ -0,0 +1,84 @@
+"""notification_preferences table
+
+Revision ID: 0e428c1bb9bc
+Revises: 70904373eab3
+Create Date: 2025-01-22 16:38:57.407898+00:00
+
+"""
+# pylint: disable=no-member, invalid-name, missing-function-docstring, unused-import, no-name-in-module
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0e428c1bb9bc"
+down_revision = "70904373eab3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "notification_preferences",
+        sa.Column(
+            "id",
+            sa.BigInteger().with_variant(sa.Integer(), "sqlite"),
```

Comment:
What does the "sqlite" variant mean?

Reply:
This is just telling alembic to use an Integer if sqlite is the backend. It automatically generated this file and I just left it because we left it in all of the previous alembic migration files.